tools/defaults.bzl

```diff
@@ -72,15 +72,22 @@ def ng_test_library(deps = [], tsconfig = None, **kwargs):
     **kwargs
   )
 
-def ng_web_test_suite(deps = [], srcs = [], static_css = [], **kwargs):
+def ng_web_test_suite(deps = [], srcs = [], static_css = [], prebuilt_theme = False,
```

Comment:
Ah, I wasn't clear in out conversation- the `prebuilt_theme` attribute should just be removed completely.

Reply:
So just manually specify the theme for each target? 

We cannot always add a theme if this macro is being used because the CDK also uses the macro.

Reply:
I was thinking that it would be fine to include it for the cdk tests too; they shouldn't be affected either way.

Reply:
I see, it's still feels a bit odd to have CDK tests use a prebuilt theme, but I agree that this makes it consistent to how the tests run w/ Gulp + themes shouldn't affect the CDK tests.

Done! please have another look 😄